mtags-interfaces/src/main/java/scala/meta/pc/VirtualFile.java

```diff
@@ -0,0 +1,18 @@
+package scala.meta.pc;
+
+/**
+ * An interface of the virtual file that contains
+ * path information and its content.
+ */
+public interface VirtualFile {
+
+    /**
+     * The path name of the virtual file.
+     */
+    String path();
+
+    /**
+     * The content of the virtual file.
+     */
+    String value();
+}
```

Comment:
Java version of `Input.VirtualFile` for defining a method that returns `Input.VirtualFile` equivalent information in `SymbolSearch.java`.